src/Http/Http.Abstractions/src/Extensions/UseMiddlewareExtensions.cs

```diff
@@ -8,7 +8,7 @@
 using System.Runtime.CompilerServices;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Http.Abstractions;
-using Microsoft.Extensions.Internal;
+using Microsoft.Extensions.DependencyInjection;
```

Comment:
Is this so we use the DI version of ActivatorUtilities? Can we remove https://github.com/dotnet/aspnetcore/blob/dad78b3671b250c671e8711e49f53b6ddf21d5e0/src/Http/Http.Abstractions/src/Microsoft.AspNetCore.Http.Abstractions.csproj#L24 then?

Reply:
@BrennanConroy, yes indeed.

Already removed ActivatorUtilities from the project
![image](https://github.com/dotnet/aspnetcore/assets/45946424/3a3fcd55-5ce2-415f-897c-42e791fee786)

Reply:
😆 whoops, just looked at the repo instead of the PR when searching for that reference

Reply:
Oops, o well, at least we're sure that it's been cleaned up!